include/SH3/system/glprogram.hpp

```diff
@@ -26,6 +27,7 @@ namespace sh3_graphics
     public:
 
         sh3_glprogram(const std::string& name) : programName(name){Load(name);};
+        sh3_glprogram(const std::string& name, std::vector<std::string> const &attribs) : programName(name){Load(name, attribs);};
```

Comment:
Return the error in an output-parameter, like it's done for `sh3_arc_file::Read*`.